internal/server/teams.go

```diff
@@ -11,18 +12,47 @@ import (
 	"github.com/pkg/errors"
 )
 
-type team struct {
-	Rank         *int     `json:"rank,omitempty"`
-	RankingScore *float64 `json:"rankingScore,omitempty"`
+// teamHandler returns a handler to get general info for a specific team
+func (s *Server) teamHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if err := s.updateTeams(r.Context()); err != nil {
+			ihttp.Error(w, http.StatusInternalServerError)
+			go s.Logger.WithError(err).Error("unable to update teams data")
```

Comment:
Can we remove all the goroutines for logging from the codebase? The time it takes doesn't matter and it just seems messy.